test/test_gpu3.cpp

```diff
@@ -6056,11 +6058,30 @@ TEST_F(NVFuserTest, FusionCastings_CUDA) {
   }
 #endif
 
+  auto convert_aten_unsupported_dtype = [](DataType dt) -> DataType {
+    if (dt == DataType::UInt) {
+      return DataType::Int;
+    } else if (dt == DataType::UInt32) {
+      return DataType::Int32;
+    }
+    return dt;
+  };
+
   for (const auto& input_type : data_types) {
-    auto tv_in = makeContigTensor(2, input_type);
+    DataType aten_input_type = convert_aten_unsupported_dtype(input_type);
```

Comment:
nit: variable name  `aten_input_type` seems strange to me. It is a type used by nvFuser but the name suggests it is a type used by aten.

Reply:
Renamed to `proxy_input_type`